### PR TITLE
Display board copy

### PR DESCRIPTION
**Pull request by Cristian-Baeza**
I was having issues pushing more commits to the first branch I made so had to create new branch and submit PR from this one. 

It took me a while to figure out how to write a test for a "puts" but I got one working for displaying the game board. Not sure if the test is written well. Let me know how bad this is.

### CODE REVIEW

File: lib/runner.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require_relative "tictactoe"
+
+new_game = Tic_Tac_Toe.new
+new_game.show_empty_board
```
Comment (td-extreme) [inline lib/runner.rb:4]: Let's move this call into TicTacToe.run_app() 
and change the this to 
```
new_game = Tic_Tac_Toe.new
new_game.run_app
```

File: lib/tictactoe.rb [hunk 2]
```diff
@@ -1,6 +1,17 @@
 class Tic_Tac_Toe
   def run_app()
     puts("Welcome to Tic-Tac-Toe")
-    "Welcome to Tic-Tac-Toe"
   end
-end
\ No newline at end of file
+
+  def show_empty_board
```
Comment (td-extreme) [inline lib/tictactoe.rb:6]: Let's change this to return a `string` representation of the board, so that our test doesn't need to look at `stdout`

**Review by td-extreme — COMMENTED** [review]
To recap from standup.  
Let's try and separate calls to `puts` from our board creating logic. This way our tests can test the board creation logic with out having to deal with stdout. 
I'm okay with us not testing the `run_app` function for now as it is the game loop function.

File: lib/tictactoe.rb [hunk 3]
```diff
@@ -1,6 +1,17 @@
 class Tic_Tac_Toe
   def run_app()
     puts("Welcome to Tic-Tac-Toe")
-    "Welcome to Tic-Tac-Toe"
```
Comment (td-extreme) [inline lib/tictactoe.rb:4]: Let's call `puts show_empty_board` here instead of in the runner

File: lib/runner.rb (hunk 1 above)
Comment (Cristian-Baeza) [reply lib/runner.rb:4]: 👍🏽

File: lib/tictactoe.rb (hunk 2 above)
Comment (Cristian-Baeza) [reply lib/tictactoe.rb:6]: 👍🏽

File: lib/tictactoe.rb (hunk 3 above)
Comment (Cristian-Baeza) [reply lib/tictactoe.rb:4]: 👍🏽

**Pull request closed without merging**